### PR TITLE
Fix AppVeyor integration

### PR DESCRIPTION
**Pull request by RolandASc**
Tested with https://github.com/miraisolutions/winsendkeys.

See https://ci.appveyor.com/project/RolandASc/winsendkeys and possibly https://codecov.io/gh/miraisolutions/winsendkeys.

Added sink-file to capture return value and query elements.

### CODE REVIEW

**Comment by RolandASc** [comment]
See also https://github.com/codecov/codecov-python/blob/master/codecov/__init__.py for reference. Note that I only cared about AppVeyor and did also not add the pull-request element to the query.

File: R/codecov.R [hunk 1]
```diff
@@ -159,6 +159,10 @@ codecov <- function(...,
 
   coverage_json <- to_codecov(coverage)
 
+  if (!quiet) {
```
Comment (jimhester) [inline R/codecov.R]: I would prefer not to change this functionality.

File: README.md [hunk 2]
```diff
@@ -73,6 +73,33 @@ after_success:
   - Rscript -e 'covr::codecov()'
 ```
 
+If you are using [Appveyor CI](http://ci.appveyor.com), and are not using
```
Comment (jimhester) [inline README.md]: This should just be something like 

```
on_success:
  - Rscript -e "covr::codecov()"
```
e.g. https://github.com/r-lib/fs/blob/3bb86eaf969b06f439970b982cdc849646332e46/appveyor.yml#L29-L30

File: R/codecov.R (hunk 1 above)
Comment (RolandASc) [reply R/codecov.R]: ok, I have taken this back out

File: README.md (hunk 2 above)
Comment (RolandASc) [reply README.md]: Simplified accordingly. Please go ahead if you would like to reword anything else in the ReadMe.

**Comment by RolandASc** [comment]
Thanks for the comments - I agree about the default use case and have committed amendments.

**Comment by jimhester** [comment]
Thanks for reworking things and for the original PR, looks good to me now!

**Pull request merged**